playground/vite.config.ts

```diff
@@ -17,6 +17,30 @@ export default defineConfig({
       ],
       addons: {
         vueTemplate: true,
+        vueDirectives: [{
```

Comment:
I feel directives should also be treated as normal `imports`. We can extract the usage of `v-xxx` and convert it to camel case based on Vue's convention.

So normally it would be `import { VMyDirective } from 'foo'` or it could be `import { MyDirective as VMyDirective } from 'foo'`.

For dts generation, I guess we could have user to define meta like `meta: { vueDirective: true }` while the others should align with normal imports syntax

Reply:
Uhmm, to detect `v-xxx` we will need the vue code, unimport should be registered before vite:vue plugin (for example).

Reply:
done: I need to fix coverage

Reply:
now directives also declared in global